components/ILIAS/Mail/classes/class.ilMailbox.php

```diff
@@ -27,13 +27,22 @@ class ilMailbox
     private readonly ilDBInterface $db;
     private readonly ilTree $mtree;
 
-    /** @var array{b_inbox: string, c_trash: string, d_drafts: string, e_sent: string, z_local : string} */
+    /** @var array{
+     *     b_inbox: string,
+     *     c_trash: string,
+     *     d_drafts: string,
+     *     e_sent: string,
+     *     z_local : string,
+     *     o_outbox: string
+     * }
+     */
     private array $default_folders = [
         'b_inbox' => 'inbox',
         'c_trash' => 'trash',
         'd_drafts' => 'drafts',
         'e_sent' => 'sent',
         'z_local' => 'local',
+        'o_outbox' => 'outbox',
```

Comment:
Please use `e_outbox` and create a database update object for the "Mail" component for ILIAS 11, which modifies all `e_sent` entries and renames them to `f_sent`.